pynblint/nb_linting.py

```diff
@@ -262,6 +262,20 @@ def cells_too_long(notebook: Notebook) -> List[Cell]:
     ]
 
 
+def detect_multiline_comment(notebook: Notebook) -> List[Cell]:
+    """check if the notebook has multiline comment.
+
+    Args:
+        notebook (Notebook): the notebook to be analyzed.
+
+    Returns:
+        bool: ``True`` if the notebook have 3 or more line of comment
+            ``False`` otherwise.
+    """
+    pattern: Pattern[str] = re.compile(r".*#.+\n#.+\n#.+\n*")
```

Comment:
This works only for 3-line comments:

1. comments of 4 or more lines are not detected, although they are even more so worth to be translated into Markdown text;
2. the threshold number of comment lines should be a configurable variable that defaults to 3. You might look at the definition of the function `notebook_too_long` for an easy example of how to do this.